ruby_programming/basic_ruby/debugging.md

```diff
@@ -103,10 +103,12 @@ p "Using p:"
 p []
 ~~~
 
-### Debugging with Pry
+### Debugging with Pry and Pry-byebug
```

Comment:
I meant for this lesson to replace `Pry` with `Pry-byebug`. I apologize for this miscommunication, I re-read my earlier comment and understand how you come to a different conclusion. `Pry` is a dependent of `Pry-byebug`, so there is no need to install both. In addition, all the references to `Pry` will need to be changed to `Pry-byebug`.

Reply:
I left the first sentence as is, as I think it is worth mentioning pry. Changed other instances of `Pry` with `Pry-byebug` though it does feel a bit redundant towards the end.